app/views/admin/deploy_groups/show.html.erb

```diff
@@ -34,7 +34,17 @@
   <div class="form-group">
     <div class="col-lg-offset-2 col-lg-10">
       <%= link_to "Edit", edit_admin_deploy_group_path(@deploy_group), class: "btn btn-primary" %>
-      <%= link_to "Deploy all projects", deploy_all_admin_deploy_group_path(@deploy_group), class: "btn btn-default" if DeployGroup.enabled? %>
+      <% if DeployGroup.enabled? %>
+        <%= link_to "Create all stages",
+                    create_all_stages_admin_deploy_group_path(@deploy_group),
+                    class: "btn btn-default",
+                    data: {
+                        method: "post",
+                        confirm: "Create all missing stages for this deploy group? All Project marked to be included in new deploy groups will get a stage for this deploy group."
```

Comment:
`All Project marked to be included in new deploy groups that have a template stage for #{environment}`